graphql-dxm-provider/pom.xml

```diff
@@ -140,6 +141,11 @@
             <version>${dataloader.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
is it really required for the unit tests, can't find any reference to com.google.* in test classes.

Reply:
let me confirm locally again, as I had a JDK 17 version issue

Reply:
Not needed